app/events/admin.go

```diff
@@ -575,7 +613,10 @@ func (a *admin) deleteAndBan(query *tbapi.CallbackQuery, userID int64, msgID int
 	}
 
 	// we allow deleting messages from supers. This can be useful if super is training the bot by adding spam messages
-	if _, err := a.tbAPI.Request(tbapi.DeleteMessageConfig{ChatID: a.primChatID, MessageID: msgID}); err != nil {
+	if _, err := a.tbAPI.Request(tbapi.DeleteMessageConfig{BaseChatMessage: tbapi.BaseChatMessage{
```

Comment:
such if with err check on a differnt lines is not my favorite thing. Probably splitting to two operations will feels better